geonode_mapstore_client/client/js/apps/gn-home.jsx

```diff
@@ -44,13 +45,11 @@ const ConnectedRouter = connect((state) => ({
     locale: state?.locale || DEFAULT_LOCALE
 }))(Router);
 
-const routes = [
-    {
-        name: 'homepage',
-        path: '/',
-        component: Home
-    }
-];
+const viewer = {
+    [appRouteComponentTypes.homeComponent]: Home
```

Comment:
please change to HomeRoute also the import
```suggestion
    [appRouteComponentTypes.homeComponent]: HomeRoute
```